devito/finite_differences/derivative.py

```diff
@@ -202,6 +202,10 @@ def _new_from_self(self, **kwargs):
         _kwargs.update(**kwargs)
         return Derivative(expr, *self.dims, **_kwargs)
 
+    @property
+    def func(self):
+        return lambda *a, **kw: self._new_from_self(expr=a[0])
```

Comment:
why are kwargs ignored? I would think that if we allow `func` we should all related `kwargs` such as `x0, ....`